app.py

```diff
@@ -0,0 +1,537 @@
+            'product': request.form.get('running_'+item["part_id"]+'_product'),
+            'link': request.form.get('running_'+item["part_id"]+'_link'),
+            'price': float(request.form.get('running_'+item["part_id"]+'_price'))
+            }
+        })
+
+    record.update({'running': running_dict})
+
+    # Adds Interior collection to record 
+    interior_dict = {}
+    for item in interior:
+        interior_dict.update({
+            item["part_id"]: {
+            'product': request.form.get('interior_'+item["part_id"]+'_product'),
+            'link': request.form.get('interior_'+item["part_id"]+'_link'),
+            'price': float(request.form.get('interior_'+item["part_id"]+'_price'))
+            }
+        })
+
+    record.update({'interior': interior_dict})
+    builds.insert_one(record)
+    flash('Build Created', category='success')
+    return redirect(url_for('builds'))
+
+# View a Record
+@app.route("/view/<build_id>")
+def view_record(build_id):
+    build = mongo.db.builds.find_one({
+        "_id": ObjectId(build_id)
+    })
+
+    builds = mongo.db.builds
+    exterior = mongo.db.exterior.find()
+    engine = mongo.db.engine.find()
+    running = mongo.db.runninggear.find()
+    interior = mongo.db.interior.find()
+
+    #Check for if user has liked the build already 
+    liked_length = int(len(build['votes']['like']['users_liked']))
+
+    liked_by = []
+
+    for x in range(0, liked_length):
+        user_in_like = build['votes']['like']['users_liked'][x]
+        liked_by.append(user_in_like)
+
+    if current_user.is_authenticated: 
+        if str(current_user.email) in liked_by: 
+            user_liked = True
+        else:
+            user_liked = False
+
+        disliked_length = int(len(build['votes']['dislike']['users_disliked']))
+
+        disliked_by = []
+
+        for x in range(0, disliked_length):
+            user_in_dislike = build['votes']['dislike']['users_disliked'][x]
+            disliked_by.append(user_in_dislike)
+
+        if str(current_user.email) in disliked_by: 
+            user_disliked = True
+        else:
+            user_disliked = False
+    else: 
+        user_liked = True
+        user_disliked = True
+
+    return render_template("view.html", build=build, exterior=exterior, engine=engine, running=running, interior=interior, user_liked=user_liked, user_disliked=user_disliked)
+
+# Edit a Record
+@app.route("/edit/<build_id>")
+@login_required
+def edit_record(build_id):
+    build = mongo.db.builds.find_one({
+        "_id": ObjectId(build_id)
+    })
+
+    exterior = mongo.db.exterior.find()
+    engine = mongo.db.engine.find()
+    running = mongo.db.runninggear.find()
+    interior = mongo.db.interior.find()
+
+    return render_template("edit.html", build=build, exterior=list(exterior), engine=list(engine), running=list(running), interior=list(interior))
+
+# Update a Record
+@app.route("/update_record/<build_id>", methods=['POST'])
+def update_record(build_id):
```

Comment:
Use get and post methods in one route instead of two routes for the functionality in this route and the previous one